.do/db-cluster/Makefile

```diff
@@ -6,7 +6,7 @@ TF ?= $(TF_EXTRA_OPTS) terraform
 WORKSPACE_PREFIX ?= postgres
 VARS_FILE ?= deploy.tfvars
 K8S_CLUSTER_REGION ?= fra1
-K8S_CLUSTER_ID ?= $(shell cat ../k8s-cluster/terraform_outputs.json | jq -r '.cluster_id.value')
+K8S_CLUSTER_ID ?= $(shell cat ../k8s-cluster/terraform_outputs.json 2>/dev/null | jq -r '.cluster_id.value' || True && echo 'REPLACE_ME')
```

Comment:
Can you think of any improvement to do here?

Reply:
how about something like this:
```
doctl kubernetes cluster get codacy-doks-cluster -o json | jq -r '.[].id'
```

Reply:
Will employ the other approach, thank you.